node.gyp

```diff
@@ -309,6 +309,27 @@
           # the executable and rename it back to node.exe later
           'product_name': '<(node_core_target_name)-win',
         }],
+        [ 'node_report=="true"', {
+          'defines': [
+            'NODE_REPORT',
+          ],
+          'conditions': [
+            ['OS=="win"', {
```

Comment:
What impact, if any, does this have on the Windows binary? Are these just dynamically linked? Do they expand the size of the binary or make it any less portable?

Reply:
@rvagg - those are just dlls, only occupy space when loaded, but not in the disk:
```gdb
$ ldd Release\node
        ntdll.dll => /c/Windows/SYSTEM32/ntdll.dll (0x77720000)
        kernel32.dll => /c/Windows/system32/kernel32.dll (0x77600000)
        KERNELBASE.dll => /c/Windows/system32/KERNELBASE.dll (0x7fefd420000)
        NETAPI32.dll => /c/Windows/system32/NETAPI32.dll (0x7fefc850000)
        netutils.dll => /c/Windows/system32/netutils.dll (0x7fefc840000)
        msvcrt.dll => /c/Windows/system32/msvcrt.dll (0x7fefeb30000)
        srvcli.dll => /c/Windows/system32/srvcli.dll (0x7fefd0f0000)
        RPCRT4.dll => /c/Windows/system32/RPCRT4.dll (0x7fefd8d0000)
        wkscli.dll => /c/Windows/system32/wkscli.dll (0x7fefc820000)
        PSAPI.DLL => /c/Windows/system32/PSAPI.DLL (0x778c0000)
        WS2_32.dll => /c/Windows/system32/WS2_32.dll (0x7fefea10000)
        NSI.dll => /c/Windows/system32/NSI.dll (0x7fefe1b0000)
        dbghelp.dll => /c/Windows/system32/dbghelp.dll (0x7fefaac0000)
        ADVAPI32.dll => /c/Windows/system32/ADVAPI32.dll (0x7fefe400000)
        sechost.dll => /c/Windows/SYSTEM32/sechost.dll (0x7fefda00000)
        USER32.dll => /c/Windows/system32/USER32.dll (0x77500000)
        GDI32.dll => /c/Windows/system32/GDI32.dll (0x7fefdb00000)
        LPK.dll => /c/Windows/system32/LPK.dll (0x7fefebd0000)
        USP10.dll => /c/Windows/system32/USP10.dll (0x7fefea60000)
        IPHLPAPI.DLL => /c/Windows/system32/IPHLPAPI.DLL (0x7fefa2c0000)
        WINNSI.DLL => /c/Windows/system32/WINNSI.DLL (0x7fefa2b0000)
        USERENV.dll => /c/Windows/system32/USERENV.dll (0x7fefd770000)
        profapi.dll => /c/Windows/system32/profapi.dll (0x7fefd3f0000)
        CRYPT32.dll => /c/Windows/system32/CRYPT32.dll (0x7fefd540000)
        MSASN1.dll => /c/Windows/system32/MSASN1.dll (0x7fefd400000)
        WINMM.dll => /c/Windows/system32/WINMM.dll (0x7fefb220000)
```

from portability perspective: it successfully ran as it is on all the supported Windows variants we have in our CI without any additional steps; that means these dlls are present by default on their default paths. So I don't foresee any portability issues.

Hoep this clarifies.